test/Emission.test.ts

```diff
@@ -0,0 +1,129 @@
+import { expect } from 'chai';
+import hre, { ethers } from 'hardhat';
+import * as mento from '@mento-protocol/mento-sdk';
+import * as helpers from '@nomicfoundation/hardhat-toolbox/network-helpers';
+
+import {
+  Emission,
+  Emission__factory,
+  MentoToken,
+  MentoToken__factory,
+} from '@mento-protocol/mento-core-ts';
+
+describe('Emission Contract', function () {
+  const { provider } = ethers;
+
+  const DAY = 60 * 60 * 24;
+  const MONTH = 30 * DAY;
+  const YEAR = 365 * DAY;
+
+  const TOTAL_EMISSION_SUPPLY = BigInt('650000000000000000000000000');
+  const EMISSION_SCHEDULE: [string, number, BigInt][] = [
+    ['1 month', MONTH, BigInt('3692586569806444700000000')],
+    ['6 months', 6 * MONTH, BigInt('21843234320275928950000000')],
+    ['1 year', YEAR, BigInt('43528555608969174900000000')],
+    ['10 years', 10 * YEAR, BigInt('325091005879265576600000000')],
+    ['15 years', 15 * YEAR, BigInt('421181077924230273650000000')],
+    ['25 years', 25 * YEAR, BigInt('554584121910167354200000000')],
```

Comment:
we can just add "n" at the end of the numbers instead of casting it to BigInt every time